development/readme.md

```diff
@@ -58,7 +55,7 @@ If you don't want to or can't use `x11docker` then you can try using simple X se
 ### Python 3
```

Comment:
It's not technically related but the Python 3 stuff is not required anymore so lets just remove that section?

Reply:
Done!